### PR TITLE
Package encoding.0.0.1

### PR DESCRIPTION
**Pull request by filipeom**
### `encoding.0.0.1`
Smt encoding library
An OCaml abstraction layer for constraint solvers.

---
* Homepage: https://github.com/formalsec/encoding
* Source repo: git+https://github.com/formalsec/encoding.git
* Bug tracker: https://github.com/formalsec/encoding/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

**Comment by filipeom** [comment]
CI failures are related with Z3 failing to compile in the macos environment.

File: packages/encoding/encoding.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+authors: ["Filipe Marques <filipe.s.marques@tecnico.ulisboa.pt>"]
+license: "GPL-3.0-only"
+homepage: "https://github.com/formalsec/encoding"
+bug-reports: "https://github.com/formalsec/encoding/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>= "5.1.0"}
+  "z3" {>= "4.11.2" & < "4.13"}
+  "menhir"
+  "ppx_inline_test" {with-test}
```
Comment (mseri) [inline packages/encoding/encoding.0.0.1/opam:15]:
```suggestion
  "zarith" {>= "1.5"}
  "ppx_inline_test" {with-test}
```
Due to
```
#=== ERROR while compiling encoding.0.0.1 =====================================#
# context              2.2.0~alpha3 | linux/x86_64 | ocaml-base-compiler.5.1.0 | pinned(https://github.com/formalsec/encoding/archive/refs/tags/v0.0.1.tar.gz)
# path                 ~/.opam/5.1/.opam-switch/build/encoding.0.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p encoding -j 47 @install
# exit-code            1
# env-file             ~/.opam/log/encoding-7-5ef3d0.env
# output-file          ~/.opam/log/encoding-7-5ef3d0.out
### output ###
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlc.opt -w -40 -w -73 -g -bin-annot -I lib/.encoding.objs/byte -I /home/opam/.opam/5.1/lib/ocaml/threads -I /home/opam/.opam/5.1/lib/ocaml/unix -I /home/opam/.opam/5.1/lib/z3 -I /home/opam/.opam/5.1/lib/zarith -intf-suffix .ml -no-alias-deps -open Encoding -o lib/.encoding.objs/byte/encoding__Z3_mappings.cmo -c -impl lib/z3_mappings.ml)
# File "lib/z3_mappings.ml", line 689, characters 14-24:
# 689 |         Real (Q.to_float @@ Z3.Arithmetic.Real.get_ratio e)
#                     ^^^^^^^^^^
# Error: Unbound value Q.to_float
# Hint: Did you mean of_float?
# (cd _build/default && /home/opam/.opam/5.1/bin/ocamlopt.opt -w -40 -w -73 -g -O3 -I lib/.encoding.objs/byte -I lib/.encoding.objs/native -I /home/opam/.opam/5.1/lib/ocaml/threads -I /home/opam/.opam/5.1/lib/ocaml/unix -I /home/opam/.opam/5.1/lib/z3 -I /home/opam/.opam/5.1/lib/zarith -intf-suffix .ml -no-alias-deps -open Encoding -o lib/.encoding.objs/native/encoding__Z3_mappings.cmx -c -impl lib/z3_mappings.ml)
# File "lib/z3_mappings.ml", line 689, characters 14-24:
# 689 |         Real (Q.to_float @@ Z3.Arithmetic.Real.get_ratio e)
#                     ^^^^^^^^^^
# Error: Unbound value Q.to_float
# Hint: Did you mean of_float?
```

**Comment by mseri** [comment]
Thanks

**Comment by mseri** [comment]
Can you please port these fixes on your main repository branch?

**Comment by filipeom** [comment]
> Can you please port these fixes on your main repository branch? No need to retag or rerelease

Done, thank you 🙏

**Comment by mseri** [comment]
Consider announcing the release on discuss.ocaml.org. There are a community category and and announcement tag for this purpose

**Pull request merged**